### ISSUE TITLE
Fails to build

### ISSUE THREAD

**Issue by acxz** [ros-melodic-arch/ros-melodic-gazebo-plugins#1]
This package fails to build because ignition-transport fails somewhere along the line.

**Comment by acxz** [comment]
Got past that issue using `libpgm-git` instead of `libpgm` for installing `zeromq` and subsequently `ignition-transport`.

However, now I am receiving the following error:
```
Scanning dependencies of target gazebo_ros_video
[ 13%] Building CXX object CMakeFiles/gazebo_ros_video.dir/src/gazebo_ros_video.cpp.o
In file included from /usr/include/gazebo-10/gazebo/rendering/Conversions.hh:25,
                 from /usr/include/gazebo-10/gazebo/rendering/Camera.hh:47,
                 from /usr/include/gazebo-10/gazebo/rendering/rendering.hh:5,
                 from /home/apatel435/.cache/yay/ros-melodic-gazebo-plugins/src/gazebo_ros_pkgs-release-release-melodic-gazebo_plugins-2.8.4-0/include/gazebo_plugins/gazebo_ros_video.h:39,
                 from /home/apatel435/.cache/yay/ros-melodic-gazebo-plugins/src/gazebo_ros_pkgs-release-release-melodic-gazebo_plugins-2.8.4-0/src/gazebo_ros_video.cpp:24:
/usr/include/gazebo-10/gazebo/rendering/ogre_gazebo.h:26:10: fatal error: OGRE/Ogre.h: No such file or directory
 #include <OGRE/Ogre.h>
          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/gazebo_ros_video.dir/build.make:63: CMakeFiles/gazebo_ros_video.dir/src/gazebo_ros_video.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:390: CMakeFiles/gazebo_ros_video.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
Error making: ros-melodic-gazebo-plugins
```

**Comment by acxz** [comment]
After an upgrade/fix to the `ogre-1.9` package, this package now builds and installs successfully.
